### ISSUE TITLE
Bit depth of 16

### ISSUE THREAD

**Issue by robsonfelix** [qmfrederik/remoteviewing#18]
Hi,

I am trying to use the VncClient class against a Tight VNC server version 2.8.8.

It gives me an exception stating the pixelFormat is invalid. When I read the raw value by looking at the source code I see thtat TightVNC is sending a bit depth of 16. The code throws an exception if it is not 24.

Can this be resolved? thoughts?

**Comment by robsonfelix** [comment]
it does connect but all I see is a black screen.

**Comment by qmfrederik** [comment]
@robsonfelix Yes, remoteviewing only supports a very limited subset of pixel formats so that's why you'd get the exception.

There's no reason it could not be fixed (TightVNC will send you a bunch of bytes and you'll need to convert it to an image).

I don't have any plans to work in this area immediately but if you're willing to investigate happy to help.

**Comment by Starlin90** [comment]
I get the same exception, i use ultravnc client and server latest version(1.2.23). VncPixelFormat.Size is static in the code and it is 16. But in VncClient.cs NegotiateDesktop method - VncPixelFormat.Decode(this.c.Receive(VncPixelFormat.Size), 0); throws that exception. Should i change any configurations? Need your help.
